### DISCUSSION TITLE
is there any way I can cycle through free or paid ProtonVPN servers if one fails to connect?

### DISCUSSION THREAD

**Discussion by onetimecontributor** [celzero/rethink-app#1788] [Q&A] [answered]
first of all, tnx for your great app that has allowed me to overcome limitation of another app I was using solely for apps tracking prevention!

I'm sorry if my question is obvious or smth, but I was not able to figure out how I can set up rethink app to cycles through three free protonVPN wireguard servers if one fails to connect. I'm piloting rethink+vpn, so I'm wondering if such thing is possible for paid protonvpn (or surfshark or any other supporting wireguard)?

please advise.

**Comment by ignoramous** [comment, marked as answer]
In the upcoming version, `v055o`, Rethink will 'load balance' among ALL active "Always-on" (*Advanced*) WireGuard profiles.

Some day, we'll implement that for *Simple* WireGuard profiles too: #1683

**Reply by onetimecontributor** [reply]
tnx, I've actually tried to enable all three free endpoints from ProtonVPN, but they were always "failing", so I thought it's something wrong with the way I've configured it. maybe it has smth to do with free tier limitations.

**Reply by ignoramous** [reply]
Rethink sometimes *may* show a WireGuard profile as *Failing* even though it really is *idle* (that is, not doing anything consequential).

`v055o` avoids showing *Failing* when WireGuard is *idle* or is in a state from which it can recover and connect to its *Peer* endpoint, if needed.
